.github/workflows/CI-pr.yml

```diff
@@ -121,13 +117,13 @@ jobs:
           toolchain: "${{ env.rust_version }}"
 
       - name: Install sccache
-        uses: taiki-e/install-action@v2.33.28
+        uses: taiki-e/install-action@v2.41.10
```

Comment:
We'd prefer to pin actions other than the official GitHub ones (those under the `actions` organization) to specific patch versions instead of only the major versions.